app/views/users/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>edit user infomation</h1>
+
+
+<%= form_for @user, url: {action: "update"} do |f| %>
+  <%= render 'shared/error_messages' %>
+  
+  <%= f.label :username %>
+  <%= f.text_field :username,value: "User"%>
+
+  <%= f.label :email %>
+  <%= f.email_field :email, placeholder: "example@example.com" %>
+
+  <%= f.label :password %>
+  <%= f.password_field :password, value: "123456789" %>
```

Comment:
- [ ] You should have the commented out forms here too, as you did in the new file. [Requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_edit.png).